client/my-sites/customer-home/cards/launchpad/intent-write.tsx

```diff
@@ -0,0 +1,103 @@
+import { useLaunchpad } from '@automattic/data-stores';
+import { Task } from '@automattic/launchpad';
```

Comment:
Nit: shouldn't this be a type import?
```suggestion
import type { Task } from '@automattic/launchpad';
```